db/memtable_list.cc

```diff
@@ -320,8 +320,9 @@ void MemTableList::RollbackMemtableFlush(const autovector<MemTable*>& mems,
   imm_flush_needed.store(true, std::memory_order_release);
 }
 
-// Record a successful flush in the manifest file
-Status MemTableList::InstallMemtableFlushResults(
+// Maybe record a successful flush in the manifest file. It might just return
+// Status::OK letting a concurrent flush to do actual the recording..
+Status MemTableList::MaybeInstallMemtableFlushResults(
```

Comment:
What do you think about the name `TryInstallMemtableFlushResults`? The name `Maybe` sounds to me that the thread **may try**, but actually it **must try**.

Reply:
Sure